src/parser/parser.yy

```diff
@@ -1866,8 +1876,10 @@ admin_job_sentence
     ;
 
 admin_job_operation
-    : KW_COMPACT { $$ = new std::string("compact"); }
-    | KW_FLUSH   { $$ = new std::string("flush"); }
+    : KW_COMPACT         { $$ = new std::string("compact"); }
```

Comment:
you can directly to use `AdminCmd`

Reply:
seems Submit command treat the command type as String